docs/routing/configuration.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Configuration
+name: configuration
+parent: Routing and Navigation
+nav_order: 3
```

Comment:
I'd suggest to make this the first sub-nav item after index: config is essential before anything :)